src/test/lib/render_test.ts

```diff
@@ -269,6 +269,12 @@ suite('render()', () => {
       assert.equal(container.querySelector('p')!.textContent, 'baz');
     });
 
+    test('does not inject text from comments with bindings', () => {
+      const t = html`<p>${'foo'}<!-- ${'baz'} ${'baz'} -->${'bar'}</p>`;
```

Comment:
We should add tests with a commented-out element with attr bindings, and a child node binding. I don't think the current solution is enough.

Reply:
Good point